Dockerfile

```diff
@@ -1,3 +1,12 @@
+# This builder stage it's only because we need a command
+# to create a symlink and reduce the size of the image
+FROM gcr.io/distroless/static-debian11:debug-nonroot as builder
+ARG TARGETARCH
+
+COPY --chown=nonroot:nonroot --chmod=0755 dist/manager_* .
```

Comment:
Can we place the manager_* files in a 'bin' subfolder or 'multi-arch' folder?

Reply:
Fixed.